doc/Advanced.md

```diff
@@ -548,6 +548,24 @@ Below the full QC Post-processing (PP) configuration structure is described. Not
 }
 ```
 
+### External tasks configuration
```

Comment:
please also add `externalTasks` directory in the section about the global overview of the QC configuration structure (a bit above).